lib/ansible/module_utils/redfish_utils.py

```diff
@@ -656,12 +673,15 @@ def set_bios_default_settings(self):
             return response
         return {'ret': True, 'changed': True, 'msg': "Set BIOS to default settings"}
 
-    def set_one_time_boot_device(self, bootdevice):
+    def set_multi_bios_default_settings(self):
+        return self.aggregate(self.set_bios_default_settings)
+
+    def set_one_time_boot_device(self, bootdevice, systems_uri):
```

Comment:
This is a "set" method called from redfish_command (which doesn't pass in the systems_uri param). So need to remove that param here and just use self.system_uris[0] below.

Reply:
Done